matlab/JSBSimInterface.h

```diff
@@ -0,0 +1,133 @@
+#ifndef JSBSIMINTERFACE_HEADER_H
+#define JSBSIMINTERFACE_HEADER_H
+
+#include "mex.h"
+#include <FGFDMExec.h>
+#include <initialization/FGInitialCondition.h>
+#include <models/FGAuxiliary.h>
+#include <models/FGPropulsion.h>
+#include <models/FGFCS.h>
+
+using namespace JSBSim;
+
+class JSBSimInterface
+{
+public:
+	JSBSimInterface(FGFDMExec *);
+	JSBSimInterface(FGFDMExec *, double dt);
```

Comment:
```suggestion
  JSBSimInterface(void);
  JSBSimInterface(double dt);
```
`FGFDMExec` instances will be now be a member of `JSBSimInterface`